testing/tests/DevExpress.ui.widgets.scheduler/integration.base.tests.js

```diff
@@ -293,9 +293,14 @@ QUnit.test("The 'max' option should be converted to Date obj before send to work
 
 QUnit.test("Scheduler should not throw an error when the details form is opened for the first time", function(assert) {
     var errorLogStub = sinon.stub(errors, "log");
+
```

Comment:
This test checks that the bug T310366 is fixed. There are no exceptions throws in this test case.
The behavior has been changed and now the warning throws in debug scripts only. It is not a breaking change